docs/content/TreeView.mdx

```diff
@@ -289,6 +289,11 @@ See [Storybook](https://primer.style/react/storybook?path=/story/components-tree
     type={`| React.ReactNode
 | (props: {isExpanded: boolean}) => React.ReactNode`}
   />
+  <PropsTableRow
+    name="label"
+    type="string"
+    description="Provide an accessible name for the leading visual. This should provide information about what the visual indicates or represents"
```

Comment:
Should we mention that if a leading or trailing visual is purely decorative that you don't need to provide a label?